providers/base/units/usb/usb.pxu

```diff
@@ -228,6 +228,7 @@ category_id: com.canonical.plainbox::usb
 plugin: shell
 _summary: Detect storage partitions on a device on the USB bus
 command:
+  echo "Searching udev for a partitioned device connected via usb"
```

Comment:
```suggestion
  echo "Searching udev for a partitioned device connected via usb..."
```

I like adding ellipsis at the end of such a sentence to notice that something else is coming afterwards (in that case, some string about USB busses, or an error message)